lib/fluent/plugin/filter_imds.rb

```diff
@@ -82,11 +82,11 @@ def filter(tag, time, record)
         record["subscriptionId"] = data["compute"]["subscriptionId"]
         record["region"] = data["compute"]["location"]
         record["resourceGroupName"] = data["compute"]["resourceGroupName"]
-        record["resourceId"] = data["compute"]["resourceId"]
+        record["azureResourceId"] = data["compute"]["resourceId"]
```

Comment:
I understand the other changes, but I don't know why we added the `azure` prefix for resourceId.

Reply:
Reverted back to resourceId as discussed offline.

Reply:
We decided to delete resourceId, correct?

Reply:
Removed :)